Dockerfile

```diff
@@ -3,6 +3,8 @@ FROM alpine:3.10
 
 RUN apk add --no-cache curl jq
 
+RUN curl --silent https://api.github.com/repos/taimos/github-action-instana-release/releases/latest | jq -r '.tag_name' > /.version
```

Comment:
IMO this should come as a build arg

Reply:
I agree but couldn't find a way to get that information into the GH action.

Reply:
Solved in https://github.com/mmanciop/github-action-instana-release/commit/e0dd716df01f5665e46821fd6273a1e465517aaa